src/components/application_manager/include/application_manager/hmi_interfaces.h

```diff
@@ -0,0 +1,35 @@
+#ifndef HMI_INTERFACES_H
+#define HMI_INTERFACES_H
+#include "interfaces/HMI_API.h"
+
+namespace application_manager {
+
+class HmiInterfaces {
```

Comment:
@LuxoftAKutsan,
It's not exactly Interfaces, but more like Interfaces provider/controller/manager.